src/libs/Navigation/AppNavigator/WorkspaceSettingsDrawerNavigator.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+// Screens
+import BaseDrawerNavigator from './baseDrawerNavigator';
```

Comment:
Let's keep these in `PascalCase` too please. If they are React components then they should use this casing.